src/wasm-interpreter.h

```diff
@@ -586,9 +602,17 @@ class ExpressionRunner : public OverriddenVisitor<SubType, Flow> {
         return value.extAddPairwiseToUI32x4();
       case TruncSatSVecF32x4ToVecI32x4:
       case RelaxedTruncSVecF32x4ToVecI32x4:
+        // TODO: We could do this only if the actual values are in the relaxed
+        //       range.
+        if (relaxedBehavior == RelaxedBehavior::NonConstant) {
+          return NONCONSTANT_FLOW;
+        }
```

Comment:
The conditions should apply to the relaxed operations but not their non-relaxed equivalents.

Reply:
Good catch... the dangers of implicit case fallback strike again. Go and Rust got this right...

Fixed.